tests/dashboard/service_test.go

```diff
@@ -150,15 +147,28 @@ func (suite *dashboardTestSuite) testDashboard(re *require.Assertions, internalP
 			break
 		}
 	}
-	args := []string{"-u", leaderAddr, "config", "set", "dashboard-address", dashboardAddress2}
-	_, err = pdctl.ExecuteCommand(cmd, args...)
+
+	input := map[string]interface{}{
+		"dashboard-address": dashboardAddress2,
+	}
+	data, err := json.Marshal(input)
 	re.NoError(err)
+	req, _ := http.NewRequest(http.MethodPost, leaderAddr+"/pd/api/v1/config", bytes.NewBuffer(data))
+	resp, err := suite.httpClient.Do(req)
+	re.NoError(err)
+	resp.Body.Close()
 	suite.checkServiceIsStarted(re, internalProxy, servers, leader)
 	re.Equal(dashboardAddress2, leader.GetServer().GetPersistOptions().GetDashboardAddress())
 
 	// pd-ctl set stop
-	args = []string{"-u", leaderAddr, "config", "set", "dashboard-address", "none"}
```

Comment:
Shall we also test it in pd-ctl?

Reply:
Yes we have [`config` test](tools/pd-ctl/tests/config/config_test.go), I'm guessing it was written that way for convenience which introduced unnecessary dependency